probs/README.md

```diff
@@ -0,0 +1,32 @@
+# probs
+
+Running multiple operations simultaneously under one application.
+
+Execute one command:
+
+```bash
+probs engine clean-call-requests --db-uri "${ENGINE_DB_URI}"
+```
+
+Run service with configuration:
+
+```bash
+probs service --config "~/.probs/config.json"
+```
+
+Config example:
+
+```json
```

Comment:
Maybe better if this is not an array of services but rather a single service.

It is more consistent with the `probs service` command, and also less nesting in configuration is better.